src/roswire/ros2/state.py

```diff
@@ -41,6 +41,12 @@ class ROS2SystemState(SystemState):
         A mapping from actions to the names of providesr of that action.
     action_clients: Mapping[str, Collection[str]]
         A mapping from actions to the names of clients of that action
+    topic_to_type: Mapping[str, str]
+        A mapping from topics to their type
```

Comment:
For clarity, we should change `to their type` to `to the name of their type`

Reply:
Done